core/src/providers/llm.rs

```diff
@@ -336,6 +336,7 @@ pub struct LLMChatRequest {
     presence_penalty: Option<f32>,
     frequency_penalty: Option<f32>,
     extras: Option<Value>,
+    version: i32,
```

Comment:
maybe cache_version?

Reply:
🤔 It's not the cache version but more the block's version actually.

Reply:
Yeah well, it's meant to be used for the cache only so might as well be explicit. No strong fell anyway. It's just that `version` alone might raise some questions to the reader that are bigger than this really is.